src/main/scala/com/ambiata/disorder/GenPlus.scala

```diff
@@ -1,9 +1,13 @@
 package com.ambiata.disorder
 
 import org.scalacheck._, Arbitrary._, Gen._
+import scala.collection.JavaConverters._
 
 object GenPlus {
 
+  def sizedInt(from: Int, to: Int): Gen[Int] =
```

Comment:
I would consider just call it `choose` for the same reason as `nonEmptyListOf` mirrors the Gen version.